opencti-platform/opencti-front/src/components/GradientButton.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import Button, { ButtonProps } from '@mui/material/Button';
+import { useTheme } from '@mui/material/styles';
+
+interface GradientButtonProps extends ButtonProps {
+  lightModeStartColor?: string;
+  lightModeEndColor?: string;
+  darkModeStartColor?: string;
+  darkModeEndColor?: string;
+}
+
+const GradientButton: React.FunctionComponent<GradientButtonProps> = ({
+  lightModeStartColor = '#001BDA',
+  lightModeEndColor = '#0FBCFF',
+  darkModeStartColor = '#0FBCFF',
+  darkModeEndColor = '#00F1BD',
+  sx,
+  ...props
+}) => {
+  const theme = useTheme();
+  const isDarkMode = theme.palette.mode === 'dark';
+  const startColor = isDarkMode ? darkModeStartColor : lightModeStartColor;
```

Comment:
You should add them in the theme (ThemeDark.ts and ThemeLight.ts) and remove all the props.